### PR TITLE
Update _compile_options.dm

### PR DESCRIPTION
**Pull request by Iamgoofball**
Lets you properly RP your wizard.

### CODE REVIEW

File: code/_compile_options.dm [hunk 1]
```diff
@@ -34,8 +34,8 @@
 
 //MINOR TWEAKS/MISC
 #define AGE_MIN				17	//youngest a character can be
-#define AGE_MAX				85	//oldest a character can be
-#define WIZARD_AGE_MIN		30	//youngest a wizard can be
+#define AGE_MAX				2000	//oldest a character can be
```
Comment (Jalleo) [inline code/_compile_options.dm]: Oldest any spaceman can be that is! Stop him!

**Comment by optimumtact** [comment]
How do you not get the current reference?

**Comment by Jalleo** [comment]
Because I am too ancient to be aware of them. (I never get told or be made aware of them)

**Comment by optimumtact** [comment]
I was talking to goof

**Pull request closed without merging**